docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,26 @@
+= Configuration
```

Comment:
Let's move this file into a directory called `ROOT/pages/references/` to stay consistent with other docs.